src/main/scala/za/co/absa/hyperdrive/trigger/api/rest/controllers/KafkaController.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.hyperdrive.trigger.api.rest.controllers
+
+import org.springframework.web.bind.annotation._
+import za.co.absa.hyperdrive.trigger.api.rest.services.KafkaService
+import java.util.concurrent.CompletableFuture
+import javax.inject.Inject
+import scala.compat.java8.FutureConverters._
+import scala.concurrent.ExecutionContext.Implicits.global
+
+@RestController
+class KafkaController @Inject()(kafkaService: KafkaService) {
+  @GetMapping(path = Array("/doesKafkaTopicExist"))
```

Comment:
I think this endpoint does not follow best practices for REST :) I think we could get inspiration from Confluent Control Center, and name it like this: `/topics/{topic-name}/exists?servers=`

Obviously this is still not the textbook REST way, but we don't want to expose any other information, so I think, an exists endpoint is ok.

Reply:
Fixed